.travis.yml

```diff
@@ -8,4 +8,4 @@ rvm:
   - 2.0.0
 script: bundle exec rake test
 env:
-  - PUPPET_VERSION="3.8.0" STRICT_VARIABLES=yes FUTURE_PARSER=yes STRINGIFY_FACTS=false
```

Comment:
why the death of `STRINGIFY_FACTS=false`? Aren't we depending on that being the case before upgrade?